app/models/game.rb

```diff
@@ -14,6 +14,30 @@ def populate!
     populate_right_white_half!
   end
 
+  def white_check?
```

Comment:
Ignore my previous comment - I made it work in a much more elegant and Rubcop compliant way :)